src/js/utils/removeComments.ts

```diff
@@ -0,0 +1,19 @@
+const COMMENT_IDENTIFIER = '//';
+
+/**
+ * Removes comment lines from a given string.
+ *
+ * This function takes a string and removes all lines that start with '//',
+ * ignoring leading and trailing whitespace. It assumes that such lines are
+ * comments and can be safely removed.
+ *
+ * @param {string} input - The input string to remove comments from.
+ *
+ * @returns {string} The input string with comment lines removed.
```

Comment:
```suggestion
 * @returns {string} the input string with comment lines removed
```

as it's not a full normal sentence

Reply:
fixed